src/constants/api.ts

```diff
@@ -1 +1,16 @@
+import { fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
 export const BASE_URL = "http://localhost:8080";
+
+export const baseQuery = fetchBaseQuery({
```

Comment:
not a constant, let's move to more meaningful path, e.g.
`src/api/baseQuery.ts`